spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/FindHidingSubClassTest.java

```diff
@@ -0,0 +1,178 @@
+package edu.umd.cs.findbugs.detect;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+
+import org.junit.jupiter.api.Test;
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+
+public class FindHidingSubClassTest extends AbstractIntegrationTest {
+    final static String BUG_TYPE = "HSBC_HIDING_SUB_CLASS";
+
+    @Test
+    public void testBadFindHidingSubClassTest() {
```

Comment:
With Junit Jupiter the `public` is unnecessary here, and it would be more consistent with the codebase. Can you please remove the `public` here and at every other tests?

Reply:
Adopted!